swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/WorkConfigurator.kt

```diff
@@ -128,16 +128,17 @@ open class WorkConfigurator(
     }
 
     // local is additive to global. local can override global
-    merge(local!!, global!!)
```

Comment:
Goddamn. @aravindmd Reason why there were so few resources being marked. The allow list of the other resource type was bleeding into other resource types.

Reply:
Old bug but now manifesting because the server group exclusions were different than that of images and snapshots.